### DISCUSSION TITLE
Adding zk to `nixpkgs`

### DISCUSSION THREAD

**Discussion by pinpox** [zk-org/zk#95] [Share]
Just wanted to let you know, I'm adding `zk` to [nixpkgs](https://github.com/NixOS/nixpkgs) so it can be installed directly on NixOS, since there seems to be interest by others in using is on that OS.

There is a PR [already open for it](https://github.com/NixOS/nixpkgs/pull/143020) and awaiting review. I'll be keeping the package up to date there, should I miss a release just ping me and I'll bump the version.

**Comment by mickael-menu** [comment]
Cool, thanks!

**Comment by keinsell** [comment]
I've installed `zk` in my `home-manager` configuration, so it seems package was actually published some time ago and discussion can be closed.
